subsys/bluetooth/controller/ll_sw/lll_sync_iso.h

```diff
@@ -55,6 +55,8 @@ struct lll_sync_iso {
 
 	uint8_t stream_curr:5;
 
+	uint8_t establish_events:3;
+
```

Comment:
Isnt this only assigned and not referenced? Do we need this? or something is missing?

Reply:
Yes, but since the supervision handling is implemented in LLL as per your design, this LLL variable is used only on vendor code for now. Will be happy to share what we did with it.

Reply:
Fair enough.
Could you mentioned the qualification test case details here? (so, that I can use it for testing when implementing for nRF SoCs).